src/reference/informative/informA2_6.rst

```diff
@@ -0,0 +1,97 @@
+                                              imported, but its id attribute "p" is not.
+                                                                 ╵
+      # In the document "CarbohydrateIdeas.cellml".              ╵
+      # All id attributes should be unique within this document. ╵
+      model: name = "Carbs" id = none specified                  ╵
+        ├─ component: name = "Chocolate", id = "c"               ╵
+        ├─ component: name = "Porridge", id = "p" ╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┘
+        └─ component: name = "Toast", id = "t"
+            └─ variable: name = "Peanutbutter", id = "p" 
+
+            # Invalid: The variable Peanutbutter is not allowed to have an id of "p"
+            # as this conflicts with the component Porridge id in the same document.
+
+
+    .. container:: toggle
+
+      .. container:: header
+
+        See CellML syntax
+
+      .. code-block:: XML
+
+        <!-- In the file BreakfastMenu.cellml: -->
+        <model name="BreakfastOfChampions" id="b">
+          <component name="Protien" id="p" />
+
+          <import href="CarbohydrateIdeas.cellml">
+            <!-- Valid: The id attribute value is set here to be "c" 
+                 rather than that of the import, "p" -->
+            <component name="Carbohydrates" id="c" component_ref="Porridge" />
+          </import>
+
+          <!-- Valid: The name attributes are repeated, but in different element types. 
+               The id attributes are distinct. -->
+          <component name="FunStuff" id="f" />
+            <variable name="FunStuff" id="fs" units="sprinkles" />
+          </component>
+        </model>
+
+        <!-- In the file CarbohydrateIdeas.cellml: -->
+        <model name="Carbs">
+          <!-- The id attribute "c" here does not clash with the Carbohydrates 
+               component above as they are unknown to each other,
+               and in different documents. -->
+          <component name="Chocolate" id="c" />
+          
+          <!-- Valid: The id attribute "p" here does not clash with the Protien 
+               component id above because even though they are in the same model,
+               they are in different documents. -->
+          <component name="Porridge" id="p" />
+
+          <component name="Toast" id="t">
+            <!-- Invalid: The id attribute "p" here does clash with the component 
+                 Porridge above because they are in the same document. -->
+            <variable name="Peanutbutter" id="p" units="smooth" />
+          </component>
+        </model>
+
+    Some points to note:
+    
+    - Here we have two separate *documents* which are partially combined to make one *model*.
+    - There is **no** repetition of :code:`id` attribute values within the BreakfastMenu.cellml *document*.
+    - There **is** repetition of the :code:`name` attribute "FunStuff", but the elements have different types (:code:`component` and :code:`variable`), so this remains valid.
```

Comment:
Similar argument here, the name _attribute_ is XML, and doesn't get imported. A component called "FunStuff" gets imported

Reply:
There's no importing of FunStuff (what a sad thing to write ...)?  They're both defined explicitly in the importing model?